content/developer/testing/develop-a-test-plan.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: practice
+weight: 5
+title: Develop a test plan
+what: >
+  A test plan exists to document how aspects of your tool will be tested. This includes automated and manual tests and how often they will be run.
+why: >
+  Having a test plan enables developers to know what needs to be tested and validated before a release or merge.
+when: Pre-Release
```

Comment:
This is harder to define in the set of `when` options you're using. Yes, typically a test plan is written for a given release. But I think we're really recommending more of a test policy; a plan that gets reused throughout the project. Additionally, as I said in another comment, you really only need to document this when things get complicated.